internal/cmd/providers.go

```diff
@@ -378,20 +378,29 @@ func GetProviderByName(client *api.Client, name string) (*api.Provider, error) {
 	return &providers.Items[0], nil
 }
 
-func parsePrivateKey(key string) (string, error) {
-	if json.Valid([]byte(key)) {
+// parsePrivateKey checks if a service account key file was passed in and parses information from it
+func parsePrivateKey(opts *providerAPIOptions) error {
```

Comment:
I feel like moving providerAPIOptions into here makes it a little less reusable, but it seems ok.